Cargo.toml

```diff
@@ -83,7 +83,7 @@ port_scanner = "0.1.5"
 rand = "0.8.5"
 regex = "1.10.5"
 reqwest = { version = "0.12.2", default-features = false, features = ["json"] }
-roaring = "0.10"
+roaring = { version = "0.10", git = "https://github.com/RoaringBitmap/roaring-rs.git" }
```

Comment:
Waiting on [this](https://github.com/RoaringBitmap/roaring-rs/pull/316/files#diff-05dc7f18d55c41aa3866ae0cbcf3ef890fbe3996872a5f3affd9879c48caa8a6R20) to be released by roaring-rs so that we don't need the git ref here